examples/jsm/loaders/GLTFLoader.d.ts

```diff
@@ -14,7 +14,9 @@ export interface GLTF {
 	scene: Scene;
 	scenes: Scene[];
 	cameras: Camera[];
-	asset: object;
+	asset: any;
```

Comment:
We might want to make it:

```suggestion
	asset: {
		copyright?: string;
		generator?: string;
		version?: string;
		minVersion?: string;
		extensions?: any;
		extras?: any;
	};
```

Note that there is no description about GLTF schema in anywhere in this repository though

Reply:
Looks good to me!